tools/batchTestGenerator/internal/config.go

```diff
@@ -56,8 +56,22 @@ type ClusterTarget struct {
 }
 
 type Target struct {
-	Name   string `json:"name"`
-	Region string `json:"region"`
+	Name            string   `json:"name"`
+	Region          string   `json:"region"`
+	ExcludedTests   []string `json:"excluded_tests"`
+	excludedTestMap map[string]struct{}
+}
+
+func (t *Target) isTestCaseExcluded(value string) bool {
+	if t.excludedTestMap == nil {
+		t.excludedTestMap = make(map[string]struct{})
+		for _, val := range t.ExcludedTests {
+			t.excludedTestMap[val] = struct{}{}
+		}
+	}
```

Comment:
This could probably be done in an `UnmarshalJSON()` method.

Reply:
Good suggestion, done!